packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -141,29 +172,51 @@ export default function LabeledField(props: Props) {
     }
 
     function maybeRenderError(): React.ReactNode | null | undefined {
-        if (!error) {
-            return null;
-        }
-
         return (
             <React.Fragment>
                 <Strut size={spacing.small_12} />
-                <LabelSmall
-                    style={light ? styles.lightError : styles.error}
-                    role="alert"
-                    id={errorId}
-                    testId={testId && `${testId}-error`}
+                <View
+                    style={styles.errorSection}
+                    id={errorSectionId}
+                    testId={testId && `${testId}-error-section`}
+                    aria-live="assertive"
```

Comment:
**question:** I wonder if we should include `aria-atomic=true` here. I'm thinking on the situation where there are more than 1 validation rules. For example, using an email validation:

1. Field is empty
2. Format is invalid
3. Using the invalid email

In cases like this, the error could persist, but the contents of the error region could change, and ussing `aria-atomic` could help Screen Readers to announce these changes in the DOM. Wdyt?

Reply:
Good call! I added in `aria-atomic=true`! I added a Story for an example where the error message changes:  https://5e1bf4b385e3fb0020b7073c-fmfsnacksw.chromatic.com/?path=/story/packages-labeledfield--changing-errors

From testing, without this set, an updated error message is still announced, however, the "Error:" prefix from the icon isn't. Setting `aria-atomic` makes it so both are announced when the error message changes :)